ucte/ucte-converter/src/main/java/com/powsybl/ucte/converter/UcteImporter.java

```diff
@@ -110,6 +112,7 @@ private static void createBuses(UcteNetworkExt ucteNetwork, UcteVoltageLevel uct
                 continue;
             }
 
+            reporter.report("createBus_" + ucteNodeCode, "Create bus ${bus}", "bus", ucteNodeCode, MarkerImpl.TRACE);
```

Comment:
Here we have a design issue. To be translated to another language, keys have to be independent of data.
For instance, here you are going to generate reports with as many key as buses.
Like  createBus_AAAAA1
Like  createBus_AAAAA2
...
How are we going to manage translation dictionnary with this kind of keys?

Reply:
With a builder style we would have:
```java
reporter.withKey("createBus").
            .withDefaultMessage("Create bus ${bus}")
            .withValue("bus", ucteNodeCode)
            .withSeverity("TRACE")
            .report();
```

Reply:
Corrected. I got confused with the unicity of keys in one reporter, but that's not needed in fact.